.github/workflows/deploy-prod.yml

```diff
@@ -1,7 +1,7 @@
 name: Deploy Prod
 
 env:
-  IMAGE_TAG: ${{ inputs.IS_HOTFIX == 'true' && format('hotfix-{0}-{1}', vars.PATCH_BRANCH, vars.HOTFIX_TAG) || github.ref_name }}
```

Comment:
This is the one spot where we don't want to change - having the ref_name here lets us use the unique tag for that release